modules/tiscamera.yaml

```diff
@@ -0,0 +1,41 @@
+name: tiscamera
+modules:
+
+- name: libzip
+  buildsystem: cmake-ninja
+  config-opts:
+    - -DCMAKE_INSTALL_LIBDIR=lib
+  cleanup:
+    - "/include"
+    - "/bin"
+    - "/share"
+    - "/lib/pkgconfig"
+    - "*.la"
+  sources:
+    - type: archive
+      url: https://libzip.org/download/libzip-1.8.0.tar.xz
+      sha256: f0763bda24ba947e80430be787c4b068d8b6aa6027a26a19923f0acfa3dac97e
+
+- name: tiscamera
+  buildsystem: cmake-ninja
+  builddir: true
+  config-opts:
+    - -DCMAKE_INSTALL_LIBDIR=lib
+    - -DTCAM_INSTALL_FORCE_PREFIX=ON
+    - -DTCAM_INSTALL_UDEV=/app/lib/udev
+    - -DTCAM_INSTALL_SYSTEMD=/app/lib/systemd/system/
+    - -DTCAM_BUILD_WITH_GUI=OFF
+    - -DTCAM_BUILD_DOCUMENTATION=OFF
+    - -DTCAM_DOWNLOAD_MESON=OFF
+    - -DTCAM_INTERNAL_ARAVIS=OFF
+    - -Daravis_INCLUDE_DIR=/app/include/aravis-0.8
+    - -DGOBJECT_INTROSPECTION_1.0_GIRDIR=/app/share/gir-1.0/
+    - -DGOBJECT_INTROSPECTION_1.0_TYPELIBDIR=/app/lib/girepository-1.0
+    - -DGSTREAMER_1.0_PLUGINSDIR=/app/lib/gstreamer-1.0
+    - -DGSTREAMER_1.0_INCLUDEDIR=/include/gstreamer-1.0
```

Comment:
Is this correct? shouldn't it be `/app/include/gstreamer-1.0` ?

Reply:
I guess it should be... It's just weird that this has never failed. This section has not been touched for a long time.

Reply:
mayeb it is not needed and can be dropped?